node/tests/GlideClient.test.ts

```diff
@@ -1396,19 +1503,23 @@ describe("GlideClient", () => {
         TIMEOUT,
     );
 
-    runBaseTests<Context>({
-        init: async (protocol, clientName?) => {
-            const options = getClientConfigurationOption(
+    runBaseTests({
+        init: async (protocol, configOverrides) => {
+            const config = getClientConfigurationOption(
```

Comment:
would you be able to do something like this? 
```
            const config = getClientConfigurationOption(
                cluster.getAddresses(),
                protocol,
                ...configOverrides
            );

```

Reply:
Excellent idea, it worked!